vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -110,29 +113,60 @@ You will need to restart VS Code after these changes. If PowerShell is still not
         }
     }
 
+    private TryFindWorkingCommand(commands : string[]) : [string, boolean]
+    {
+        for(const command of commands)
+        {
+            try
+            {
+                const cmdFoundOutput = cp.spawnSync(command);
+                if(cmdFoundOutput.status === 0)
+                {
+                    this.eventStream.post(new DotnetAltnerativeCommandFoundEvent(`The command ${command} was found.`));

```

Comment:
nit: typo here - `DotnetAltnerativeCommandFoundEvent` should be `DotnetAlternativeCommandFoundEvent`